telegram/ext/_application.py

```diff
@@ -911,6 +912,11 @@ def run_webhook(
                 header isn't set or it is set to a wrong token.
 
                 .. versionadded:: 20.0
+            unix (:class:`pathlib.Path` | :obj:`str`, optional): Path to the unix socket file. Path
+                can be empty in which case the file will be created. When using this param, you
```

Comment:
```suggestion
                does not need to exist, in which case the file will be created. When using this param, you
```

"can be empty" sounds like "you can pass an empty string '' here" to me 🤔

Reply:
Depending on the os I bet you can